app/controllers/Subscriptions.scala

```diff
@@ -147,20 +153,32 @@ class Subscriptions(
 
   def premiumTierGeoRedirect: Action[AnyContent] = geoRedirect("subscribe/premium-tier")
 
-  def displayForm(countryCode: String, displayCheckout: String, isCsrf: Boolean = false): Action[AnyContent] =
-    authenticatedAction(recurringIdentityClientId) { implicit request =>
+  private def digitalSubscriptionFormHtml(idUser: IdUser, countryCode: String)(implicit request: RequestHeader, settings: Settings): Html = {
+    val title = "Support the Guardian | Digital Subscription"
+    val id = "digital-subscription-checkout-page-" + countryCode
+    val js = "digitalSubscriptionCheckoutPage.js"
+    val css = "digitalSubscriptionCheckoutPageStyles.css"
+    val csrf = CSRF.getToken.value
+
+    digitalSubscription(title, id, js, css, Some(csrf), idUser)
+  }
+
+  def displayForm(countryCode: String, displayCheckout: String, isCsrf: Boolean = false): Action[AnyContent] = {
```

Comment:
can we make `displayCheckout` a `Boolean`?

Reply:
I asked @Ap0c about why it's a String and it's because there's no need to make it possible for a type check to fail when we just need the param to hide the page 
(a la
![image](https://user-images.githubusercontent.com/3072877/49869088-3c9dc980-fe07-11e8-9485-c2e28318b1d0.png) )

But apparently we've had this conversation on the team before so I'm going to change it to a Boolean anyway 😅